src/linclust/kmermatcher.cpp

```diff
@@ -321,6 +328,36 @@ std::pair<size_t, size_t> fillKmerPositionArray(KmerPosition<T> * kmerArray, siz
                             threadKmerBuffer[bufferPos].id = seqId;
                             threadKmerBuffer[bufferPos].pos = (kmers + kmerIdx)->pos;
                             threadKmerBuffer[bufferPos].seqLen = seq.L;
+                            // store adjacent seq information
+                            unsigned int startPos = (kmers + kmerIdx)->pos;
+                            unsigned int endPos = (kmers + kmerIdx)->pos + adjustedKmerSize - 1;
+                            for (size_t i = 0; i < 6; i++) {
+                                if (TYPE == Parameters::DBTYPE_NUCLEOTIDES) {
+                                    threadKmerBuffer[bufferPos].adjacentSeq[i] = static_cast<unsigned char>(4);
+                                }else {
+                                    threadKmerBuffer[bufferPos].adjacentSeq[i] = static_cast<unsigned char>(12);
```

Comment:
These should not be hardcoded, i guess these should be the `X`/`N` residue?

Normally we do:
```
const unsigned char xIndex = m->aa2num[static_cast<int>('X')];
```

Reply:
Fixed it